src/express-lane-auction/ExpressLaneAuction.sol

```diff
@@ -5,16 +5,14 @@ import "./Errors.sol";
 import {Balance, BalanceLib} from "./Balance.sol";
 import {SafeERC20, IERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import {ECDSA} from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
-import {
```

Comment:
Should probably include the formatting changes as a separate point on the release notes

Reply:
sgtm